<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerrverDialog</class>
 <widget class="QDialog" name="SerrverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Server Information</string>
   </property>
   <widget class="QLabel" name="lbl_port_number">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>7777</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ip_addr">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>256:256:256:256</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_listen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start listening</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connections:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_connection_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>155</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBx_chat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>501</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Chat</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Message for client</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_msg_to_client">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_send_mesg">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Message from client:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_msg_from_client">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a client from the table</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_close">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
